src/Cfonb240Reader.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the CFONB Parser package.
+ *
+ * (c) Guillaume Sainthillier <hello@silarhi.fr>
+ * (c) @fezfez <demonchaux.stephane@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Silarhi\Cfonb;
+
+use Silarhi\Cfonb\Banking\Header;
+use Silarhi\Cfonb\Banking\Total;
+use Silarhi\Cfonb\Banking\Transaction;
+use Silarhi\Cfonb\Banking\Transfer;
+use Silarhi\Cfonb\Exceptions\ParseException;
+use Silarhi\Cfonb\Parser\Cfonb120\EmptyParser;
```

Comment:
EmptyParser should be moved to a neutral namespace as it's used by every parsers (120 & 240)

Reply:
done